packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/instance/leftFlanking..st

```diff
@@ -0,0 +1,4 @@
+accessing-private
+leftFlanking: aBoolean
```

Comment:
Hmm, I don't know what guidelines are saying, but it feels strange that this is named differently.
Couldn't setter and instance variable be named `isLeftFlanking`?

Same for `isRightFlanking:`.

Reply:
I don't understand what you mean :D

Reply:
I mean it is weird that the getter is `isLeftFlanking` (which I like), but instance variable and setter have the shorter form `leftFlanking`. Like I said, I have no idea what guidelines are saying, when in doubt, ask at our next meeting

Reply:
Well usually for boolean setter you would use `beLeftFlanking` and `beNotLeftFlanking` but then you would need both and that's kind of weird. Not sure why the best option is

Reply:
We decided to keep it